### PR TITLE
Mention new trs-notebook example in Notebooks update

### PR DESCRIPTION
**Pull request by kathy-t**
I created a notebook that demonstrates how to use the TRS API in [this repository](https://github.com/dockstore/dockstore-trs-notebook/blob/main/trs-notebook.ipynb) and registered it to Dockstore. 

This PR mentions this notebook in the News And Updates section in the existing Notebooks news entry.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-6079

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]

File: news.html [hunk 1]
```diff
@@ -17,7 +17,9 @@ <h5 class="mb-0">
              href="https://docs.dockstore.org/en/stable/getting-started/getting-started-with-notebooks.html">
               Notebooks</a>
       </h5>
-        <span>Learn how to register notebooks in Dockstore</span>
+        <span>Learn how to register notebooks in Dockstore. Explore an <a target="_blank" rel="noopener noreferrer"
```
Comment (denis-yuen) [inline news.html:20]: Maybe link the two sentences or transition in some way, like "as a fun example" ...

Comment (denis-yuen) [reply news.html:20]: Basically don't want users to confuse the TRS example with learning how to register (in the header above)

**Review by denis-yuen — APPROVED** [review]

**Comment by kathy-t** [comment]
Re-requesting reviews for more discussion about the TRS notebook example. Please view it here https://dockstore.org/notebooks/github.com/dockstore/dockstore-trs-notebook:main?tab=info

**Review by coverbeck — APPROVED** [review]
After you merge this, and check it looks good on staging, you could just go ahead and merge develop to master to it will appear in prod, if you have permissions.

**Review by svonworl — APPROVED** [review]

File: news.html [hunk 2]
```diff
@@ -17,7 +17,9 @@ <h5 class="mb-0">
              href="https://docs.dockstore.org/en/stable/getting-started/getting-started-with-notebooks.html">
               Notebooks</a>
       </h5>
-        <span>Learn how to register notebooks in Dockstore</span>
+        <span>Learn how to register notebooks in Dockstore. For a fun example of a notebook on Dockstore, explore this <a target="_blank" rel="noopener noreferrer"
```
Comment (denis-yuen) [inline news.html:20]: Hmmm, was thinking about more of a discussion. 
Do we have a different tutorial notebook that we can point to so we can describe that before describing the more complex TRS notebook example?  
We could even consider two news items one for registration and a simple notebook example, and a second one for the TRS notebook.

Comment (denis-yuen) [reply news.html:20]: This is follow-up for https://oicr.slack.com/archives/C05EZH3RVNY/p1707412377242599

**Comment by kathy-t** [comment]
Created a follow up ticket to create a simpler notebook https://ucsc-cgl.atlassian.net/browse/SEAB-6274 that can be included in the news entry. I'm going to close this PR for now since the TRS notebook is too complex of an example.

**Comment by denis-yuen** [comment]
@kathy-t was this linked in the documentation repo? 
I think we still need this to be findable in some way, if only for our own sanity when preparing presentations

**Pull request closed without merging**